src/IssueViz.Security.UnitTests/IssuesStoreToIssueLocationStoreAdapterTests.cs

```diff
@@ -0,0 +1,297 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Composition.Hosting;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarLint.VisualStudio.Core.Analysis;
+using SonarLint.VisualStudio.Integration.UnitTests;
+using SonarLint.VisualStudio.IssueVisualization.Editor.LocationTagging;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using SonarLint.VisualStudio.IssueVisualization.Security.IssuesStore;
+using IssuesChangedEventArgs = SonarLint.VisualStudio.IssueVisualization.Editor.LocationTagging.IssuesChangedEventArgs;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests
+{
+    [TestClass]
+    public class IssuesStoreToIssueLocationStoreAdapterTests
+    {
+        [TestMethod]
+        public void MefCtor_CheckExports()
+        {
+            var batch = new CompositionBatch();
+
+            batch.AddExport(MefTestHelpers.CreateExport<IIssuesStore>(Mock.Of<IIssuesStore>()));
+            batch.AddExport(MefTestHelpers.CreateExport<IIssuesStore>(Mock.Of<IIssuesStore>()));
+            batch.AddExport(MefTestHelpers.CreateExport<IIssuesStore>(Mock.Of<IIssuesStore>()));
```

Comment:
The test isn't checking whether these types are actually imported so they aren't adding any value (in fact, we'd expect the export of the aggregating adapter to work even if there were no exported `IIssuesStores`).

I think there are two separate things to check:
1) is the class exportable as an `IIssueLocationStore`? (as commented elsewhere, I'm not sure the other export is required)
2) does the class import multiple `IIssuesStore` instances? (not currently tested)